app/code/Magento/Review/Ui/DataProvider/Product/ReviewDataProvider.php

```diff
@@ -58,14 +58,22 @@ public function __construct(
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      * @since 100.1.0
      */
     public function getData()
     {
         $this->getCollection()->addEntityFilter($this->request->getParam('current_product_id', 0))
             ->addStoreData();
 
+        $params = $this->request->getParams();
+        if (isset($params['sorting'])) {
+            $sorting = $this->request->getParam('sorting');
```

Comment:
Please check a more simplified version that should also work.

```suggestion
        if ($sorting = $this->request->getParam('sorting', null)) {
```